### PR TITLE
Switch to custom PB TC

### PR DESCRIPTION
**Pull request by NiLuJe**
c.f. https://github.com/koreader/koxtoolchain/pull/22

Got rid of all the GCC 4.8 patches.
Got rid of the dedicated toolchain target.
Stopped flagging PB as LEGACY (??!!), because that made no sense in the first place anyway.

Will probably need a rebase because I caught https://github.com/koreader/koreader-base/pull/1047 while working on this, so it's currently in here too ;).

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader-base/1048)

### CODE REVIEW

**Comment by NiLuJe** [comment]
NOTE: I'm not quite sure why the glib shared library is shipped on PB?

I *think*/*hope* that we're linking sdcv statically... (We might not have historically always been, since Kindle ships a glib).

File: Makefile [hunk 1]
```diff
@@ -87,7 +87,7 @@ $(OUTPUT_DIR)/data:
 
 libs: \
 	$(if $(or $(SDL),$(ANDROID)),,$(OUTPUT_DIR)/libs/libkoreader-input.so) \
-	$(if $(or $(SDL),$(ANDROID),$(KINDLE),$(KOBO),$(POCKETBOOK)),$(OUTPUT_DIR)/libs/libblitbuffer.so,) \
+	$(if $(or $(CERVANTES),$(KINDLE),$(KOBO),$(POCKETBOOK),$(REMARKABLE),$(SONY_PRSTUX)),$(OUTPUT_DIR)/libs/libblitbuffer.so,) \
```
Comment (Frenzie) [inline Makefile]: Did SDL and Android just disappear? :-P

Comment (NiLuJe) [reply Makefile]: Oh, crap. :D. (I c/p pasted the zsync2 lines :D).

**Comment by NiLuJe** [comment]
(Rebased without #1047)

File: Makefile (hunk 1 above)
Comment (NiLuJe) [reply Makefile]: Fixed in #1047, thanks ;).

**Pull request merged**